pallets/gear-debug/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub mod pallet {
     #[pallet::error]
     pub enum Error<T> {}
 
+    /// Program debug info.
+    /// TODO: unfortunatelly we cannot store pages data in [PageBuf],
```

Comment:
TODO line must be comment, not doc string

Reply:
Done, make self debug

Reply:
Done